### ISSUE TITLE
Donated, still no points

### ISSUE THREAD

**Issue by Leebus1** [dalaranwow/dalaran-wow#4876]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I purchased points, but my bank flagged it as suspicious.  PaymentWall stated it was denied, then I got a text from my bank asking if it was valid.  I replied YES, which allowed me to then submit again.  There are currently 2 charges, with one being reversed.  

**Expected behaviour**: I should have 29 points.

**Steps to reproduce the problem**:

1.  Submit payment
2. Denied
3. Approved transaction and submitted again

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Leebus1 if this issue is not resolved already use **dalaran.community@gmail.com** as the bug tracker is not for these kind of problem.
